src/css/application.css

```diff
@@ -342,6 +349,52 @@ body {
   }
 }
 
+/** @define instructions-editor */
+
+.instructions-editor {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  background-color: var(--color-low-contrast-gray);
+}
+
+.instructions-editor__input {
+  padding: 1rem 1rem 0;
+  flex-grow: 1;
+  overflow-y: scroll;
+
+  /* postcss-bem-linter: ignore */
+  & textarea {
```

Comment:
Any reason to break out of BEM here? We control the `<textarea>` so we could just give it an element class…